src/docs/sdk/event-payloads/properties/contexts_trace.mdx

```diff
@@ -0,0 +1,9 @@
+`contexts.trace`:
+
+: _Required_. Contains all the data relevat for the Transaction/Span.
```

Comment:
```suggestion
: _Required_. Contains all the data relevant for the Transaction/Span.
```

Reply:
What is the purpose of context? Are there fields that are recommended to be included?

Reply:
Yeah they have to be there and "represent" the Span